gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StringPaths.java

```diff
@@ -169,4 +172,14 @@ static String toDirectoryPath(String path) {
         ? path
         : path + GoogleCloudStorage.PATH_DELIMITER;
   }
+
+  /**
+   * @param bucketName name of the bucket
+   * @return bucket name in to GCS v2 bucket name format
+   */
+  static String toV2BucketName(String bucketName) {
```

Comment:
`StringPaths` class intended for helper methods that manipulate a full GCS object path, but this seems to be very gRPC wire protocol specific. May we abstract this in some gRPC base class if we have it, or create gRPC API helper class and put it there?